continuous_integration/recipes/distributed/meta.yaml

```
@@ -44,7 +44,7 @@ outputs:
       script: >
         python -m pip install . -vv --no-deps
         --install-option="--with-cython=profile"  # [cython_enabled]
-      track_features: {{ "cythonized-scheduler" if cython_enabled else "" }}
+      track_features: "[cythonized-scheduler]"    # [cython_enabled]
```

Would suggest formatting like this for clarity (regarding the `list` point above)

```suggestion
      track_features:             # [cython_enabled]
        - cythonized-scheduler    # [cython_enabled]
```